src/faq.md

```diff
@@ -21,34 +21,41 @@ No, it does not. TLS is designed to guarantee the authenticity of data **only to
 
 The TLSNotary protocol overcomes this limitation by making the third-party `Verifier` a participant in the TLS connection. 
 
-### How exactly does a Verifier participate in the TLS connection? { #faq2 }
+### Why is it necessary to add a verifier to the TLS connection? { #faq2 }
+
+One may wonder why the `Prover` can not simply generate a proof of the TLS connection locally without the help of another party.
+
+This is not possible because of the way TLS is designed. Specifically, TLS utilizes symmetric-key cryptography with message authentication codes (MACs). As a consequence the TLS client, i.e. the `Prover`, 
+knows the secret key the `Server` uses to authenticate data and can trivially generate fake transcripts locally. Introducing another party into the connection mitigates this problem.
```

Comment:
```suggestion
knows the secret key the `Server` uses to authenticate data and can trivially generate fake transcripts locally. Introducing another party into the connection mitigates this problem, because the secret key is now split between the client and this additional party.
```